src/screens/ContactScreen/index.js

```diff
@@ -185,6 +198,12 @@ const ContactScreen = ({navigation}) => {
     }
   };
 
+  const handleAddParticipant = () => {
+    const newSelectedUsers = selectedUsers;
+    setSelectedUsers([]);
+    onAddMember(newSelectedUsers);
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleAddParticipant` function is added. It uses the `onAddMember` function from the `useGroupInfo` hook to add new participants. Ensure that the `onAddMember` function handles the case when `newSelectedUsers` is an empty array.
```diff
+   const handleAddParticipant = () => {
+     const newSelectedUsers = selectedUsers;
+     setSelectedUsers([]);
+     onAddMember(newSelectedUsers);
+   };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->